src/engine.rs

```diff
@@ -775,7 +773,13 @@ impl<'a, 'b> Cpu<'a, 'b> {
                 let word_offs = tcdm_addr - 4 * word_addr;
                 let ptr: *const u32 = self.tcdm_ptr[id];
                 let word = unsafe { *ptr.offset(word_addr as isize) };
-                (word >> (8 * word_offs)) & ((((1 as u64) << (8 << size)) - 1) as u32)
+                let val = (word >> (8 * word_offs)) & ((((1 as u64) << (8 << size)) - 1) as u32);
+                debug!(
```

Comment:
Can you please change the debugging level to `trace` here as it was before?

Reply:
Addressed in [this commit](https://github.com/pulp-platform/banshee/pull/18/commits/95619028583909da22c571e6fdbf3cd400d35545).